src/kubelet/kubelet_test.go

```diff
@@ -84,6 +91,53 @@ func TestScraper(t *testing.T) {
 	}
 }
 
+func TestScraper_FilterNamespace(t *testing.T) {
+	// We test with a specific version to not modify number of entities
+	version := testutil.Version(testutil.Testdata122)
+
+	t.Run(fmt.Sprintf("for_version_%s", version), func(t *testing.T) {
+		testServer, err := version.Server()
+		if err != nil {
+			t.Fatalf("Cannot create fake kubelet server: %v", err)
+		}
```

Comment:
Nit: as testify is already been used, I'd change the error-check ifs by `require.NoError`

Reply:
I've done it with require.noError in this new tests but didn't modify it for the old integration tests since the errors are wrapped and I am not sure if adding this context is really meaningful